### DISCUSSION TITLE
Any companies other than Square using this in production yet?

### DISCUSSION THREAD

**Discussion by mikedtwc** [square/workflow-kotlin#345] [Q&A]
Do y'all know if any other companies are using Workflow in production, aside from square? I've just started looking at it and going through the tutorial, am really liking what I see, and we're strongly considering using it. Some folks are understandably shy to do so with it still in Alpha, and if other companies are using Workflow already, it may ease their minds a bit. Thanks!

**Comment by rjrjr** [comment]
@SalvatoreT is using it at [Persona](https://withpersona.com/). I know other folks in our public Slack are kicking the tires, but I don't know who has shipped what.

**Comment by rjrjr** [comment]
Our friends at [Careem](https://www.careem.com/) have been using it for a while.

**Comment by tudorluca** [comment]
Shift STUDIO is building [ok.com](https://www.ok.com/)'s Android Skylight app with 100% Workflows & Jetpack Compose (no androidx.ViewModel or View.java).

**Comment by Jparrgam** [comment]
[Hapi](https://hapi.trade) we are migrating to 100% workflow, we have a module in production with workflow and it works perfectly
